tests/web_api/test_parameters.py

```diff
@@ -18,9 +18,13 @@ def test_return_code():
 
 def test_response_data():
     parameters = json.loads(parameters_response.data.decode('utf-8'))
+
     assert_equal(
         parameters['taxes.income_tax_rate'],
-        {'description': 'Income tax rate'}
+        {
+            'description': 'Income tax rate',
+            'href': 'http://localhost/parameter/taxes.income_tax_rate'
```

Comment:
We want `'http://localhost/parameter/taxes/income_tax_rate'` here as it's the new standard route to access a parameter